ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -369,4 +370,76 @@ private Schema buildAggregateSchema(
     return schemaBuilder.build();
   }
 
+  private List<Expression> getInternalExpressionList(
+      final List<Expression> expressionList,
+      final InternalSchema internalSchema
+  ) {
+    return expressionList.stream()
+        .map(argExpression -> argExpression instanceof Literal
+                              ? argExpression
+                              : new QualifiedNameReference(
+            QualifiedName.of(
+                internalSchema
+                    .getExpressionToInternalColumnNameMap()
+                    .get(argExpression.toString()))))
+        .collect(Collectors.toList());
+  }
+
+  private List<Pair<String, Expression>> updateFinalSelectExpressions(
```

Comment:
I think this can also be method within `InternalSchema`. It is directly using the `expressionToInternalColumnNameMap` which is owned by the `InternalSchema`.

Perhaps name it something like `InternalSchema.updateFinalSelectExpressions(List<Pair<String, Expression>> finalSelectExpressions)`. 

In general, I think encapsulating these details in something which is semantically easier to understand has great value for someone looking at the code for the first time. Seeing a bunch of maps and lists passed around makes the intent of the code very hard to grok.